Add view header (#37813)
home-assistant/home-assistant.github.io: source/_dashboards/sections.markdown

   - You are now presented with a new, empty view.
   - If you chose a background image, the page is filled with that image.

Once you have created a sections view, you can populate it sections and cards. You can also add a header with a title and badges.

## Editing the header

<p class='img'>
  <img src="/images/dashboards/sections_view_header_edit.png" alt="Editing the header"/>
  Editing the header
</p>

1. To add a title, select the **Add title** button. The title supports [Markdown](https://commonmark.org/help/) and [templating](/docs/configuration/templating/).
2. To add badges, elect the **Add badge** button. Follow [steps on adding badges](/dashboards/badges) to see the different possible options.

> 3. To change the title and badges disposition, select the edit {% icon "mdi:edit" %} button to access header settings.

Reviewer: ```suggestion
3. To change the title and badges disposition, select the edit {% icon "mdi:edit" %} button to access header settings.

```

  description: "`sections`"
  type: string
{% endconfiguration %}

## Header YAML configuration

{% configuration %}
layout:
  required: false
  description: Layout of the different elements. Can be `start`, `center` or `responsive`. `responsive` will be the same as `start` on mobile device and will place badges and title side by side on desktop.
  type: string
  default: center
badges_position:
  required: false
  description: Badges position. Can be `bottom` or `top`.
  type: string
  default: bottom
card:
  required: true

>   description: Card to be used as title. If you are configuring the view using the visual editor, it will use the configuration of the [markdown card](/dashboards/markdown).

Reviewer: ```suggestion
  description: Card to be used as title. If you are configuring the view using the visual editor, it will use the configuration of the [Markdown card](/dashboards/markdown).
```

  description: "`sections`"
  type: string
{% endconfiguration %}

## Header YAML configuration

{% configuration %}
layout:
  required: false

>   description: Layout of the different elements. Can be `start`, `center` or `responsive`. `responsive` will be the same as `start` on mobile device and will place badges and title side by side on desktop.

Reviewer: ```suggestion
  description: Layout of the different elements. Can be `start`, `center`, or `responsive`. `responsive` is the same as `start` on mobile devices. It places badges and title side by side on desktop.
```

  type: string
  default: center
badges_position:
  required: false
  description: Badges position. Can be `bottom` or `top`.
  type: string
  default: bottom
card:
  required: true

>   description: Card to be used as title. If you are configuring the view using the visual editor, it will use the configuration of the [Markdown card](/dashboards/markdown).

Reviewer: ```suggestion
  description: Card to be used as title. If you are configuring the view using the visual editor, the configuration of the [Markdown card](/dashboards/markdown) is used.
```
